content/source/docs/enterprise-beta/sentinel/enforce.html.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: enterprise2
+page_title: "Sentinel - Terraform Enterprise Beta"
+sidebar_current: "docs-enterprise2-sentinel-enforce"
+---
+
+# Enforce and Override Policies
+
+Once a policy is added to an organization it is enforced on all subsequent Workspace Runs.
+
+The policy check will occur immediately after a plan is successfully executed in the run. If the plan fails, the policy check will not be performed. The policy check uses the tfplan generated in the plan of the run along with the state and configuration to verify the rules in each of the policies.
+
+All hard mandatory and soft mandatory policies must pass in order for the run to continue to the the Confirm and Apply state. A failed advisory policy will allow the run to continue.
```

Comment:
should we define these policy levels in this section?

Reply:
or link to manage-policies

Reply:
I will link them to the manage-policies page.

I was thinking that we might want to move it from manage-policies to this section. However, this section focuses on the consumer and how the policy is enforced on their workspace. The manage-policies section focuses on the producer user who sets the policies. The enforcement level seems something that is more relevant for the producer in this context, so I think it makes sense to keep it in the manage-policies page.

Reply:
nice sounds good!